p4-16/psa/psa.p4

```diff
@@ -78,7 +78,7 @@ const   CloneSessionId_t PSA_CLONE_SESSION_TO_CPU = unspecified;
  * fields may be used in a single P4 header definition, even on P4
  * implementations that restrict headers to contain fields with a
  * total length that is a multiple of 8 bits. */
-typedef bit<16> PortIdInHeader_t;
+typedef bit<32> PortIdInHeader_t;
```

Comment:
This change was suggested by Waqar Mohsin in a P4-API working group meeting, and several others agreed it was a good change to make.